platform.txt

```diff
@@ -20,7 +20,7 @@
 # https://github.com/arduino/Arduino/wiki/Arduino-IDE-1.5---3rd-party-Hardware-specification
 
 name=Adafruit SAMD (32-bits ARM Cortex-M0+ and Cortex-M4) Boards
-version=1.5.14
+version=1.5.15
```

Comment:
can you change this to 1.6.0 - its a fairly big change!

Reply:
Done